wiki/entry_schmitz.qmd

```diff
@@ -0,0 +1,45 @@
+--
+title: "Wiki Entry Template"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: Ronja Schmitz 
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+Children and Adolescents and Digital Media 
+
+Key Quote: 
+
+“Thus, digital media can provide an engaging experience in which the media experiences of children and teenagers become highly personalized.“
+
+Argument
+ 
+Der Text befasst sich insbesondere mit den Risiken der (online) Mediennutzung für Kinder und Jugendliche. Es werden die verschiedenen Altersstufen betrachtet und welche individuellen Gefahren die Mediennutzung für die jeweilige Altersgruppe darstellen kann. Es wird an dieser Stelle betont, dass man davon ausgehen muss, dass ebenfalls Persönlichkeit und die sozialen Umstände eine wichtige Rolle dabei spielen, inwiefern die Nutzung zur Gefahr wird. 
```

Comment:
The text deals in particular with the risks of (online) media use for children and young people. It looks at the different age groups and what individual dangers media use can pose for each age group. It is emphasized at this point that it must be assumed that personality and social circumstances also play an important role in the extent to which use becomes a danger.